odiglet/debug.Dockerfile

```
@@ -117,10 +99,7 @@ COPY --from=public.ecr.aws/odigos/agents/nodejs-community:v0.0.8 /instrumentatio
 COPY --from=dotnet-builder /dotnet-instrumentation /instrumentations/dotnet
 
 # PHP
-COPY --from=php-agents /php-agents/opentelemetry-php/8.1 /instrumentations/php/8.1
-COPY --from=php-agents /php-agents/opentelemetry-php/8.2 /instrumentations/php/8.2
-COPY --from=php-agents /php-agents/opentelemetry-php/8.3 /instrumentations/php/8.3
-COPY --from=php-agents /php-agents/opentelemetry-php/8.4 /instrumentations/php/8.4
+COPY --from=public.ecr.aws/odigos/agents/php-community:v0.2.4 /instrumentations/php /instrumentations/php
```

nit (for ease-of-read-and-maintain):

```suggestion
ARG PHP_AGENT_VERSION="v0.2.4"
COPY --from=public.ecr.aws/odigos/agents/php-community:${PHP_AGENT_VERSION} /instrumentations/php /instrumentations/php
```